aeron-client/src/main/c/aeron_agent.c

```diff
@@ -282,6 +283,19 @@ aeron_idle_strategy_t aeron_idle_strategy_backoff =
         aeron_idle_strategy_backoff_state_init_args
     };
 
+static const aeron_symbol_table_obj_t aeron_idle_strategy_table[] =
+    {
+        { "sleeping", "aeron_idle_strategy_sleeping", &aeron_idle_strategy_sleeping },
```

Comment:
Name should be `sleep` rather than `sleeping` to have consistent naming.

Reply:
I agree, but I'd like to make that a separate change to this PR.  The current PR preserves behaviour: https://github.com/real-logic/aeron/blob/master/aeron-client/src/main/c/aeron_agent.c#L301.  I think it also requires a "sleep-ms" to be consistent with the Java driver as well.